security/tor/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=          tor
-PLUGIN_VERSION=       1.2
+PLUGIN_VERSION=       1.3
```

Comment:
we don't need this unless you don't want these changes release in 17.7.8

Reply:
why not - It does not harm anything

Reply:
because now we are at 1.1 and then we are at 1.3 :P

Reply:
no problem, then 1.2 will be a developer preview.